src/website/src/pages/ec/components/card/docs/usage.md

```diff
@@ -3,10 +3,55 @@ title: Usage
 order: 1
 ---
 
-The cards component serves as a container that groups various information and interface elements. They are an entry point to more detailed information and are grouped on the grid.
+import { Link, Paragraph } from '@ecl/website-components';
 
-They can be formatted in landscape or portrait.
+<Paragraph size="lead">
+  Cards act as an entry point to more detailed information. A card is container for a few short, related pieces of information. It roughly resembles a playing card in size and shape, and is intended as a linked, short representation of a conceptual unit.
+</Paragraph>
 
-Cards are grouped on the grid, and can be formatted in landscape or portrait.
+## Anatomy
 
-More information will be uploaded, work in progress.
+Card layouts can vary to support the types of content they contain.
+
+| Name        | Mandatory |                     Note |
+| ----------- | :-------: | -----------------------: |
+| Thumbnail   |           | SHOULD BE CLICKABLE !!!! |
+| Title       |     X     | SHOULD BE CLICKABLE !!!! |
+| Description |           |                          |
+| Meta        |           |                          |
+| Tags        |           |                          |
+| Infos       |           |                          |
+
+## When to use
+
+- For information browsing (as opposed to searching)
+- To group heterogeneous types of content
+- To provide a visual boundary, allowing users to easily navigate through:
+  - several distinct items
+  - To enhance information browsing
+- The user goals that the card-based web design best responds to:
+  - scrolling through
+  - scanning through
+  - browsing through
+
+## When not to use
+
+- For information searching (as opposed to browsing).
+- On small screen display avoid using too many cards as they force users to scroll down and rely on their short-term memory. This creates a cognitive overload which harms UX.
+
+## DO's
+
+- Use cards to emphasize on contents
+- Only display the most relevant information on each card
+  illustration: card with a thumbnail, title, description
+- Keep the amount of links on the cards as low as possible
+
+## DON'Ts
+
+- Overuse cards; Limit the amount of cards to small groups of item.
```

Comment:
```suggestion
- Use of cards should be limited to 3-4 within a row, not to be mixed up as list items
```

Reply:
The suggested change is a nice addition but shouldn't replace the one in place.

Reply:
I replied it.